contracts/crowdsale/ZxcCrowdsale.sol

```diff
@@ -204,9 +204,13 @@ contract ZxcCrowdsale is
     uint256 tokens;
 
     // Sender needs Xcert KYC token.
-    require(xcertKyc.balanceOf(beneficiary) > 0);
-
+    uint256 balance = xcertKyc.balanceOf(msg.sender);
+    require(balance > 0);
+    
     if (isPresale()) {
+      uint256 tokenId = xcertKyc.tokenOfOwnerByIndex(beneficiary, balance.sub(1));
```

Comment:
We do expect for every user to just have one KYC token, right? Maybe we should be more strict - balance should be == 1 and token index is then 0.

Reply:
Even tho we do expect that everyone will have only 1 KYC token there is a possibility that something unexpected happens. So this is just a precaution. 

@xpepermint What do you think?